src/Modal/Modal.component.tsx

```diff
@@ -30,32 +28,18 @@ export type Props = ReactModal.Props & {
    **/
   modalTitle: string;
   /**
-   * Specify the function to execute when the user clicks Cancel or No.
-   * The handler should care of closing the modal, e.g. changing the `isOpen` prop.
+   * Specify the function to execute when the user clicks the title close button.
+   * The close button on the title will be unavailable if this function is not available.
    *
    * @default null
    **/
-  onCancelOrNo?: any;
-  /**
-   * Specify the function to execute when the user clicks OK or Yes.
-   * The handler should care of closing the modal, e.g. changing the `isOpen` prop.
-   *
-   * @default null
-   **/
-  onOkOrYes?: any;
-  /**
-   * Any props that should be passed directly to the third-
-   * party react-modal control.
-   *
-   * @default null
-   **/
-  controlSpecificProps?: any;
+  onTitleClose?: any;
```

Comment:
I suppose the propType for onTitleClose is probably be `onTitleClose?(e: React.MouseEvent<HTMLElement>): void;`?